shellwhat/parsers.py

```diff
@@ -23,3 +20,19 @@ class OshTransformer(NodeTransformer):
     def visit_Sentence(self, node):
         return node.child
 
+class DummyParser(AstModule):
+    def parse(self, *args, **kwargs):
+        return None
+
+# Set defaults for customizing how which parser is used,
+# and how OSH is called
+import os
+parse_opt = os.environ.get('SHELLWHAT_PARSER')
+
+DEFAULT_PARSER = DummyParser if parse_opt == '0' else OshParser
+print(DEFAULT_PARSER)
```

Comment:
Is this a leftover debug statement or should it be here every time?

Reply:
It's there to allow people to disable the parser, or opt in to use the parser via docker.

see https://github.com/datacamp/shellwhat/pull/28/files/ba11b6b68bde93d1a157268c7e6da30c2359a2ff#diff-04c6e90faac2675aa89e2176d2eec7d8R36

(the parser is built into the base shell image on datacamp, so this is more to ease local development)